thorlcr/activities/enth/thenthslave.cpp

```diff
@@ -82,8 +83,8 @@ class BaseEnthActivity : public CSlaveActivity, implements ILookAheadStopNotify
         else
         {
             localRecCount = RCUNBOUND;
-            IStartableEngineRowStream *lookAhead = createRowStreamLookAhead(this, originalInputStream, queryRowInterfaces(input), ENTH_SMART_BUFFER_SIZE, true, false, RCUNBOUND, this, &container.queryJob().queryIDiskUsage());
-            setLookAhead(0, lookAhead); // NB: this is post base start()
+            lookAhead = createRowStreamLookAhead(this, originalInputStream, queryRowInterfaces(input), ENTH_SMART_BUFFER_SIZE, true, false, RCUNBOUND, this, &container.queryJob().queryIDiskUsage());
+            replaceInputStream(0, lookAhead); // NB: this is post base start()
```

Comment:
You're right, previous method used to capture the lookahead in an owned, now links it only.
Will fix.